package.json

```diff
@@ -19,8 +19,8 @@
     "playwright-aws-lambda": "^0.7.0",
     "playwright-core": "^1.17.1",
     "prism-react-renderer": "^1.2.1",
-    "react": "^0.0.0-experimental-42c30e8b1-20220104",
-    "react-dom": "^0.0.0-experimental-42c30e8b1-20220104",
```

Comment:
Here I assumed I was using an experimental release for a reason, but looking at the project imports– I don't think that's the case. The newest hook I use is `useSyncExternalStore` which was released in 18.0